docs/checkit.js

```diff
@@ -460,8 +460,8 @@ CheckIt.prototype.generateCheckmarks = function(calObj, $calendarDiv) {
 
 CheckIt.prototype.removeEmptyWeeks = function(calObj, $calendarDiv) {
     // Remove empty weeks from the calendar
-        
-    $calendarDiv.find('.month > .week').each( function(index) {
+    
+    $calendarDiv.find('.month > tbody > .week').each( function(index) {
```

Comment:
Okay, if I just leave it as '.month . week' it works. This is how you select a descendant. (not necessarily a direct child).